python/tests/expressions/test_expressions_base.py

```diff
@@ -208,9 +208,9 @@ def test_notnan_bind_nonfloat():
 @pytest.mark.parametrize(
     "op, string",
     [
-        (base.And(ExpressionA(), ExpressionB()), "And(testexpra, testexprb)"),
-        (base.Or(ExpressionA(), ExpressionB()), "Or(testexpra, testexprb)"),
-        (base.Not(ExpressionA()), "Not(testexpra)"),
+        (base.And(ExpressionA(), ExpressionB()), "And(left=ExpressionA(), right=ExpressionB())"),
+        (base.Or(ExpressionA(), ExpressionB()), "Or(left=ExpressionA(), right=ExpressionB())"),
+        (base.Not(ExpressionA()), "Not(child=ExpressionA())"),
```

Comment:
nit: since updating this file, maybe worth try simply LINE 220 to 245 as sam suggested in https://github.com/apache/iceberg/pull/5362/files#r932825540

```python
def test_ref_binding_case_sensitive(request):
    schema = request.getfixturevalue("table_schema_simple")
```

can be replaced by python
```
def test_ref_binding_case_sensitive(table_schema_simple):
```

Reply:
I didn't make that change because I think it is easier to understand what's happening when we use less magic. Using `getfixturevalue` makes it clear that this is getting a fixture called "table_schema_simple". Otherwise where that comes from is magic and confusing (at least to me).